### PR TITLE
#BAH-912 | Export multiple forms in form builder 

### PR DESCRIPTION
**Pull request by AlekhyaYalla**
User should be able to export multiple forms from form builder

### CODE REVIEW

**Comment by bsneha90** [comment]
Can be merged after 'https://github.com/Bahmni/openmrs-module-bahmni.ie.apps/pull/5#pullrequestreview-337039662' is fixed.

File: src/common/constants.js [hunk 1]
```diff
@@ -6,4 +7,9 @@ export const commonConstants = {
   saveSuccessMessage: 'Form Saved Successfully',
   publishSuccessMessage: 'Form Successfully Published',
   toastTimeout: 4000,
+  logPath: '/var/log/openmrs/openmrs.log',
```
[the remark this answers is not in the document]
Comment (bsneha90) [reply src/common/constants.js]: It actually comes from 'log.location' setting of openmrs.

File: src/form-builder/components/FormList.jsx [hunk 2]
```diff
@@ -23,6 +23,11 @@ export default class FormList extends Component {
     const data = this._getUpdatedFormList(this.props.data);
     return map(data, (rowItem, index) => (
       <tr key={rowItem.id}>
+        <td className="form-list-check-box">
+          { rowItem.published ? <input onChange={() => this.props.handleSelectedForm(rowItem) }
+            type="checkbox"
```
Comment (bsneha90) [inline src/form-builder/components/FormList.jsx]: we can just write it like "rowItem.published && <input onChange={() => this.props.handleSelectedForm(rowItem) }
            type="checkbox"'/>", ternary operator not required

File: src/common/constants.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (AlekhyaYalla) [reply src/common/constants.js]: We need to make an api call to the database to get that location from openmrs global properties. So as it is just for display purpose we gave that location in the constants file

File: src/form-builder/components/FormList.jsx (hunk 2 above)
Comment (AlekhyaYalla) [reply src/form-builder/components/FormList.jsx]: yes changed

File: src/common/constants.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (bsneha90) [reply src/common/constants.js]: In case someone is changing it, the display message will be wrong right?

**Pull request merged**